### ISSUE TITLE
Custom template tags

### ISSUE THREAD

**Issue by fabianscheidt** [awsmug/torro-forms#457]
## Issue Overview

I'm trying to write a custom template tag with some content that I'll define later but I can't register the template tag properly and can't find proper documentation for it. The group and the tag do not show up and the callback does not get called when I enter the tag manually. What am I doing wrong?

## Steps to Reproduce
Here is the code of my plugin. I kept it simple for this question. The function `register_extended_torro_templatetags_form()` gets called properly and I can also fetch the template tag handler after registering it but it does not show up in the UI.

```php
<?php
/*
Plugin Name: Extend Torro Tags
Description: Adds additional response tags to Torro Forms
Author: Fabian Scheidt
Version: 1.0
*/

namespace awsmug\Torro_Forms\Modules\Actions;

use awsmug\Torro_Forms\DB_Objects\Forms\Form;
use awsmug\Torro_Forms\DB_Objects\Submissions\Submission;
use awsmug\Torro_Forms\Components\Template_Tag_Handler;

function register_extended_torro_templatetags_form() {
    $groups = array(
        'extendedformtags' => _x( 'My Custom Extension', 'Extended Form Templatetags', 'torro-forms' ),
    );

    $tags = array(
        'mycustomtag' => array(
            'group'       => 'extendedformtags',
            'label'       => __( 'My Custom Tag', 'torro-forms' ),
            'description' => __( 'Is just a test.', 'torro-forms' ),
            'callback'    => function( $form, $submission ) {
                return 'Test successful';
            }
        )
    );

    $template_tag_handler = new Template_Tag_Handler('extendedformtags', $tags, array(Form::class, Submission::class), $groups);
    torro()->template_tag_handlers()->register($template_tag_handler);
}

function init_register_extended_torro_templatetags_form() {
    if ( function_exists( 'torro_load' ) ) {
        torro_load( __NAMESPACE__ . '\\register_extended_torro_templatetags_form' );
    } else {
        add_action( 'torro_loaded', __NAMESPACE__ . '\\register_extended_torro_templatetags_form', 10, 1 );
    }
}

add_action('init', __NAMESPACE__ . '\\init_register_extended_torro_templatetags_form', 9);
```

**Comment by mahype** [comment]
Hi Fabian,

the plugin in Version 1.0.0 does not support template tag extension. This will come in 1.1.0 and have to be done by filters you can see here:

https://github.com/awsmug/torro-forms/blob/1.1.0/src/modules/actions/email-notifications.php#L483

Sorry for the inconvenience! We are working at the 1.1.0 right now and hopefully will release it in the next weeks.

Greetings,

Sven

**Comment by fabianscheidt** [comment]
Hi Sven,

thanks for your quick response and also for your hard work building this awesome plugin ;-)
Is that feature already working in the current version of 1.1.0? I would then start to look at it now.

Thanks,
Fabian

**Comment by mahype** [comment]
Yes it is already implemented in the 1.1.0 branch. But be careful, it is our current development branch. ;)
